src/inspector_agent.cc

```diff
@@ -567,12 +578,30 @@ void AgentImpl::WaitForDisconnect() {
                            v8::ReadOnly).FromJust();                          \
   } while (0)
 
+void AddCommandLineAPIMethod(const v8::FunctionCallbackInfo<v8::Value>& info) {
+  auto env = Environment::GetCurrent(info);
+
+  if (info.Length() != 2 || !info[0]->IsString() || !info[1]->IsFunction()) {
+    return env->ThrowError("inspector.addCommandLineAPIMethod takes exactly "
```

Comment:
I thought so but it was using `ThrowError` for the same thing above. Should I change both or just my copy?